app/assets/javascripts/autocomplete_checkboxes.js

```diff
@@ -0,0 +1,90 @@
+function hitEnter(event, element, formType, inputType) {
+  if (event.which === 13) {
+    enterOnAutocomplete(element, formType, inputType);
```

Comment:
Yes, that is correct. So they can begin to type their new quick create content and then that will open the modal with the name field pre-populated.